src/registry.json

```diff
@@ -2097,8 +2097,8 @@
         ]
       },
       "versions": {
-        "0.1.2": {
-          "checksum": "lezRrq2JT1SFQidFBWZymGnod20s23CCGsF8sW1ZmbQ="
+        "0.1.3": {
```

Comment:
What is the diff for the version change?

Reply:
Its just the version on the snap manifest apparently that changed the checksum no other changes: https://github.com/InTaxDev/fintax-snap/commit/a671a4f56d93fd3e8f105c5f02682cf780b2e004